src/System.Reflection.Metadata/src/System/Reflection/Metadata/MetadataReader.cs

```diff
@@ -96,7 +95,7 @@ public unsafe MetadataReader(byte* metadata, int length, MetadataReaderOptions o
             this.Block = new MemoryBlock(metadata, length);
 
             _options = options;
-            this.Utf8Decoder = utf8Decoder;
+            this.StringDecoder = utf8Decoder;
```

Comment:
With that said, we decided to use the UTF8 spelling (all caps in violation of design guidelines) to match the rest of FX. I think this should be `UTF8Decoder`.